### PR TITLE
PHPCS: Comment Likes Module

### PR DESCRIPTION
**Pull request by sdixon194**
This PR updates the comment likes module to meet phpcs standards.

#### Changes proposed in this Pull Request:

* This PR updates the comment likes module to meet phpcs standards.

#### Jetpack product discussion

N/a

#### Does this pull request change what data or activity we track or use?

Nope.

#### Testing instructions:

* Check out branch
* Make sure you can still enable comment likes under Discussion > Comments.
* Check out comments, make sure you can see likes, how many there are, view the like table in the comments dashboard.

### CODE REVIEW

**Comment by sdixon194** [comment]
As of right now...this PR isn't working. Renaming the file as PHPCS requires doesn't surface the module in the Jetpack discussion settings and I still haven't figured out why. I can't find anywhere else that references the php file directly.

**Comment by kraftbj** [comment]
For php files directly under the modules dir, we should just add a phpcs:ignore for the file name rule. 

The module "slug" that is used everywhere including the options of enabled modules etc is the file name minus `.php` so we really can't rename these files without a lot of work. It isn't worth it for PHPCS IMO.

**Comment by sdixon194** [comment]
> For php files directly under the modules dir, we should just add a phpcs:ignore for the file name rule.

Works for me!

File: projects/plugins/jetpack/modules/comment-likes.php [hunk 1]
```diff
@@ -9,6 +9,8 @@
  * Auto Activate: No
  * Module Tags: Social
  * Additional Search Queries: like widget, like button, like, likes
+ *
+ * @package Jetpack
```
Comment (kraftbj) [inline projects/plugins/jetpack/modules/comment-likes.php:13]:
```suggestion
 * @package automattic/jetpack
```

Moving to this format.

File: projects/plugins/jetpack/modules/comment-likes.php [hunk 2]
```diff
@@ -105,16 +118,22 @@ function enqueue_admin_styles_scripts() {
...
-			JETPACK__VERSION
+			JETPACK__VERSION,
+			$in_footer = false
 		);
 	}
 
+	/**
+	 * Adds like count column to admin page.
+	 *
+	 * @param array $columns - column of admin table.
```
Comment (kraftbj) [inline projects/plugins/jetpack/modules/comment-likes.php:129]: The dash isn't needed in phpdocs but I don't think it hurts to leave them.

**Review by kraftbj — CHANGES_REQUESTED** [review]
One minor suggestion re package name in the file docblock.

File: projects/plugins/jetpack/modules/comment-likes.php (hunk 1 above)
Comment (sdixon194) [reply projects/plugins/jetpack/modules/comment-likes.php]: Nice catch! Should be updated now.

File: projects/plugins/jetpack/modules/comment-likes.php (hunk 2 above)
Comment (sdixon194) [reply projects/plugins/jetpack/modules/comment-likes.php:129]: Yeah, I can never remember if it's JS or PHP that we need the dash so my brain just does it for both now.

File: projects/plugins/jetpack/modules/comment-likes.php [hunk 3]
```diff
@@ -105,16 +118,22 @@ function enqueue_admin_styles_scripts() {
...
+	 * */
```
Comment (anomiex) [inline projects/plugins/jetpack/modules/comment-likes.php:130]: I'm a little surprised phpcs doesn't complain about this.
```suggestion
	 */
```

Looks like you did it in a few other places in here too.

File: projects/plugins/jetpack/modules/comment-likes.php (hunk 2 above)
Comment (anomiex) [reply projects/plugins/jetpack/modules/comment-likes.php:129]: Personally I'd avoid the dash in PHP code, to match all the rest of the PHP code.

File: composer.lock [hunk 4]
```diff
@@ -466,28 +466,28 @@
         },
         {
             "name": "phpcompatibility/phpcompatibility-paragonie",
-            "version": "1.3.0",
+            "version": "1.3.1",
```
Comment (anomiex) [inline composer.lock:469]: While I don't object to upgrading the versions in composer.lock, maybe that should be a separate PR?

Comment (sdixon194) [reply composer.lock]: Oh nice catch, I think I upgraded and committed by mistake without realizing what branch I was on. That was clumsy of me.

File: projects/plugins/jetpack/modules/comment-likes.php [hunk 5]
```diff
@@ -105,16 +128,24 @@ function enqueue_admin_styles_scripts() {
 				'modules/comment-likes/comment-like-count.js'
 			),
 			array( 'jquery' ),
-			JETPACK__VERSION
+			JETPACK__VERSION,
+			$in_footer = false
```
Comment (jeherve) [inline projects/plugins/jetpack/modules/comment-likes.php:132]:
```suggestion
			false
```

File: composer.lock [hunk 6]
```diff
@@ -470,24 +470,24 @@
             "source": {
                 "type": "git",
                 "url": "https://github.com/PHPCompatibility/PHPCompatibilityParagonie.git",
-                "reference": "b862bc32f7e860d0b164b199bd995e690b4b191c"
```
Comment (jeherve) [inline composer.lock:473]: The changes to this file seem unrelated. Could you remove them?

**Review by jeherve — CHANGES_REQUESTED** [review]

File: composer.lock (hunk 6 above)
Comment (sdixon194) [reply composer.lock]: I think I goofed this up, and then when trying to restore, accidentally also updated the composer.lock in the jetpack plugin's directory. What's the best way to fix this one before I dig a deeper hole?

Comment (anomiex) [reply composer.lock]: The easiest way to reset the files is to check out the versions of the files from the merge-base of your branch and master
```
git checkout $(git merge-base HEAD master) composer.lock projects/plugins/jetpack/composer.lock
```

Comment (sdixon194) [reply composer.lock]: Beautiful thank you, should be fixed now!

**Review by anomiex — APPROVED** [review]
Seems straightforward enough.

**Review by kraftbj — APPROVED** [review]
